components/Home.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+const api = axios.create({
```

Comment:
This is great to have used but I would also look into extracting it and the other axios requests  you need to use into a separate file and then you only need to import the functions into your components when you need to make an API call. This would make your code a lot cleaner